sof_wrapper/auth/views.py

```diff
@@ -153,23 +154,20 @@ def authorize():
     # todo: define fetch_token function that requests JSON (Accept: application/json header)
     # https://github.com/lepture/authlib/blob/master/authlib/oauth2/client.py#L154
     token_response = oauth.sof.authorize_access_token(_format='json')
-    extra = {}
     extracted_id_token = extract_payload(token_response.get('id_token'))
     username = extracted_id_token.get('preferred_username')
     DEA = extracted_id_token.get('DEA')
 
     # standalone uses profile
     if 'profile' in extracted_id_token:
-        extra['user'] = extracted_id_token['profile']
+        session['user'] = extracted_id_token['profile']
     else:
-        extra['user'] = {'username': username, 'DEA': DEA}
-
-    # retain extracted user details in session, as needed elsewhere
-    session['user'] = extra['user']
+        session['user'] = {'username': username, 'DEA': DEA}
 
     if 'patient' in token_response:
-        extra['subject'] = 'Patient/{}'.format(token_response['patient'])
-    current_app.logger.info("login", extra=extra)
+        session['subject'] = 'Patient/{}'.format(token_response['patient'])
```

Comment:
Should you also look for `launch_token_patient` ie
`get_session_value('token_response', {}).get('patient') or get_session_value('launch_token_patient')`?

Reply:
we look there and capture in /launch - any reason it would be different here in /authorize ?